### PR TITLE
Created a service provider for Welwyn Hatfield Borough Council, UK.

### PR DESCRIPTION
**Pull request by tevers200**
Service provider created for Welwyn Hatfield Borough Council, UK.

Limitation in that the council only shows the next collection for each bin type, not all future collections. This is due to limitations with the WelHat council website.

# Output of tests:
```
Testing source welhat_gov_uk ...
100080965745
AL9 5EA
C:<redacted>\venv\lib\site-packages\urllib3\connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.welhat.gov.uk'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
  found 4 entries for test 1 - South Red
100080977050
AL7 3ET
C:\<redacted>\venv\lib\site-packages\urllib3\connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.welhat.gov.uk'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
  found 4 entries for test 2 - Blue North
```

# Updated files:
.py, .md, README and info.md files have all been updated
Process finished with exit code 0

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+class Source:
+    def __init__(self, uprn: str, postcode: str):
+        self._uprn = uprn
+        self._postcode = postcode
+
+    def fetch(self):
+        print(self._uprn)
+        print(self._postcode)
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py:29]: Please remove print statements

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+        response = requests.post('https://www.welhat.gov.uk/xfp/form/214', headers=headers, data=data, verify=False)
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py:62]: is verify=False needed?
I get valid data without it.

Comment (tevers200) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py]: I've created a new commit removing that - it was only required whilst testing the configuration and intercepting traffic.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py (hunk 1 above)
Comment (tevers200) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py]: I've created a new commit removing the print statements.

**Comment by tevers200** [comment]
> Service provider created for Welwyn Hatfield Borough Council, UK.
> 
> Limitation in that the council only shows the next collection for each bin type, not all future collections. This is due to limitations with the WelHat council website.
> 
> # Output of tests:
> ```
> Testing source welhat_gov_uk ...
> 100080965745
> AL9 5EA
> C:<redacted>\venv\lib\site-packages\urllib3\connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.welhat.gov.uk'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
>   warnings.warn(
>   found 4 entries for test 1 - South Red
> 100080977050
> AL7 3ET
> C:\<redacted>\venv\lib\site-packages\urllib3\connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.welhat.gov.uk'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
>   warnings.warn(
>   found 4 entries for test 2 - Blue North
> ```
> 
> # Updated files:
> .py, .md, README and info.md files have all been updated Process finished with exit code 0

New tests after removing print statements and removing the verify=False flag

**Comment by 5ila5** [comment]
Thanks for your contribution

**Pull request merged**